src/App.tsx

```diff
@@ -33,7 +32,6 @@ const App: React.FC = () => {
                 <Route path="" element={<Login />} />
                 <Route path="login/*">
                     <Route path="" element={<Login />} />
-                    <Route path="help" element={<LoginHelp />} />
```

Comment:
Still leaving the ability to nest in case we need it for our proper auth implementation